src/components/organism/PoolSelectTokenModal/index.tsx

```diff
@@ -30,7 +30,7 @@ const PoolSelectTokenModal: FC<PoolSelectTokenModalProps> = ({
     setIsModalOpen(false);
   };
 
-  const handlePoolAssetTokeId = (id: string, assetSymbol: string, decimals: string, assetTokenBalance: string) => {
+  const handlePoolAssetTokeData = (id: string, assetSymbol: string, decimals: string, assetTokenBalance: string) => {
```

Comment:
It is not a blocker but I strongly suggest that when you have e.g.: 4 properties in the function, make an object instead of demanding each of them. Eg: handlePoolAssetTokeData = (data: {id: string, assetSymbol: string, decimals: string, assetTokenBalance: string}) => 
and everywhere where use that function u will say handlePoolAssetTokeData({id: 'dsadad', assetSymbol: 'dasda'}), etc... So you do not need to worry about the order of properties that you have to pass